### ISSUE TITLE
After installation, the "nvm" command isn't available.

### ISSUE THREAD

**Issue by almet** [derekstavis/plugin-nvm#3]
After running `omf install nvm`, the command isn't available.

[![asciicast](https://asciinema.org/a/dsy5r3f4qqkqfvr21npnim1ei.png)](https://asciinema.org/a/dsy5r3f4qqkqfvr21npnim1ei)

Anything I can do?

**Comment by almet** [comment]
Oh, even after a `refresh` it's not there.

**Comment by derekstavis** [comment]
Thanks for the report!! Have you installed `nvm` via official installer? Can you please paste the output of `ls ~/.nvm`?

**Comment by derekstavis** [comment]
ping @almet

**Comment by edrpls** [comment]
Ran into the same problem, here's the output of ls ~/.nvm

```
CONTRIBUTING.md
LICENSE.md
Makefile
README.markdown
bash_completion
install.sh
nvm-exec
nvm.sh
package.json
test
```

**Comment by bangedorrunt** [comment]
I could confirm that if you install `nvm` from `brew`, it is not gonna work.

**Comment by derekstavis** [comment]
When installing from brew you need to adjust `NVM_DIR` environment variable, pointing to where brew installs it.

**Comment by 3cp** [comment]
If you are one of the Mac users like me who installed nvm through brew, and had created a local ~/.nvm folder, do this:
1. in `~/.config/fish/config.fish`, add line `set -gx NVM_DIR "/Users/YOUR_USER_NAME/.nvm"`
2. in `~/.nvm` folder, run `ln -s (brew --prefix nvm)/nvm.sh nvm.sh`

`refresh`, Good luck.

**Comment by derekstavis** [comment]
Should be solved with the merge of #5, thanks everyone for reporting and @huochunpeng for the PR. If you happen to find the same problem, please open another issue reporting. Thanks everyone :tada:!
